src/marqo/tensor_search/models/search.py

```diff
@@ -59,13 +60,42 @@ def add_content(self, content: List[Union[str, List[str]]]) -> VectorisedJobPoin
             end_idx=len(self.content)
         )
 
+
 class SearchContextTensor(BaseModel):
     vector: List[float]
     weight: float
 
 
+class SearchContextDocumentsParameters(BaseModel):
+    tensorFields: Optional[List[str]] = None
```

Comment:
One comment I got from @Yihan is that we use `tensor_fields: Optional[List[str]] = Field(None, aliase="tensorFields")` in the model so we can keep the snake case in python but also accept camelCase in API.